src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -89,10 +94,11 @@ public function downloadAction(Request $request, $id)
 
     /**
      * @param FileVersion $fileVersion
+     * @param string $dispositionType
      *
      * @return StreamedResponse
      */
-    protected function getFileResponse($fileVersion)
+    protected function getFileResponse($fileVersion, $dispositionType = 'attachment')
```

Comment:
@chirimoya mentioned that here should be a validation! i think there is only `inline` and `attachment`!